internals/cli/export_test.go

```diff
@@ -20,9 +20,14 @@ import (
 	"github.com/canonical/pebble/client"
 )
 
-var RunMain = Run
+func RunMain() error {
+	_, ClientConfig.Socket = getEnvPaths()
+	return Run(&RunOptions{
+		ClientConfig: ClientConfig,
+	})
+}
 
-var ClientConfig = &clientConfig
+var ClientConfig = &client.Config{}
```

Comment:
Would be nice to get rid of this global at some point, but probably shouldn't be in this PR.

Reply:
PR #388 now includes removal of the global.